### PR TITLE
dino: new port in net

### PR DESCRIPTION
**Pull request by barracuda156**
#### Description

New port: https://dino.im

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 14.5
Xcode 15.4

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL in commit message? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: net/dino/Portfile [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+checksums           rmd160  4c19280920647390ea8a9de7b8e2a150bfb73341 \
+                    sha256  432d7c3b5170c595b1b31a8d64d73ded26e32af9f03a2d1a01828c22a8ade3fa \
+                    size    883755
+github.tarball_from archive
+
+# FIXME: on 10.6 it builds and launches, but trying to register an account results in an error:
+# (dino:74601): Pango-WARNING **: 23:01:36.818: Invalid UTF-8 string passed to pango_layout_set_text()
+
+depends_build-append \
+                    bin:cmake:cmake \
```
Comment (reneeotten) [inline net/dino/Portfile:24]: why can this not use the "cmake" PortGroup?

Comment (barracuda156) [reply net/dino/Portfile:24]: It uses a standard configure and handles CMake from there.

Comment (reneeotten) [reply net/dino/Portfile:24]: okay, so upstream is a little confused about life 😉

Comment (barracuda156) [reply net/dino/Portfile:24]: Yeah LOL

**Pull request merged**